php-checks/src/main/java/org/sonar/php/checks/utils/CheckUtils.java

```diff
@@ -384,6 +384,11 @@ public static boolean isAbstract(ClassDeclarationTree tree) {
     return Optional.empty();
   }
 
+  public static Optional<LiteralTree> resolvedArgumentLiteral(FunctionCallTree call, String name, int position) {
```

Comment:
(it's just me, but) I have no idea what this method is doing and how it is doing it.